.github/workflows/quick-test.yml

```diff
@@ -23,71 +25,73 @@ jobs:
+          - driver: bazel
+            run-test: cd c++ && bazel test --verbose_failures --test_output=errors //...
+          - driver: super-test
+            run-test: ./super-test.sh quick
+          - clang: 12
+            # clang-12 on Ubuntu requires libunwind to be installed explicitly
+            libunwind: libunwind-12-dev
     steps:
       - uses: actions/checkout@v2
+      - uses: bazelbuild/setup-bazelisk@v2
       - name: install dependencies
+        # I observed 404s for some packages and added an `apt-get update`. Then, I observed package
+        # conflicts between LLVM 14 and 15, and added the line which removes LLVM 14, the default on
+        # Ubuntu as of this writing.
         run: |
             export DEBIAN_FRONTEND=noninteractive
-            sudo apt-get install -y build-essential git zlib1g-dev cmake libssl-dev ${{ matrix.compiler }}
-      - name: super-test
-        run: |
-            ./super-test.sh quick ${{ matrix.compiler }}
+            sudo apt-get update
+            sudo apt-get remove clang-14 libc++1-14 libc++abi1-14 libc++-14-dev libc++abi-14-dev libunwind-14 libunwind-14-dev
+            sudo apt-get install -y build-essential git zlib1g-dev cmake libssl-dev clang-${{ matrix.clang }} libc++-${{ matrix.clang }}-dev libc++abi-${{ matrix.clang }}-dev ${{ matrix.libunwind }}
+      - name: test
+        run: |
+            export CC=clang-${{ matrix.clang }}
+            export CXX=clang++-${{ matrix.clang }}
+            ${{ matrix.run-test }}
   Linux-lock-tracking:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        compiler: [clang-9]
+        compiler: [clang-14]
         features: ["-DKJ_TRACK_LOCK_BLOCKING=1 -DKJ_SAVE_ACQUIRED_LOCK_INFO=1 -DKJ_CONTENTION_WARNING_THRESHOLD=200"]
     steps:
       - uses: actions/checkout@v2
       - name: install dependencies
         run: |
             export DEBIAN_FRONTEND=noninteractive
             sudo apt-get install -y build-essential git zlib1g-dev cmake libssl-dev ${{ matrix.compiler }}
-      - name: super-test
+      - name: super-test ${{ matrix.compiler }}
         run: |
             # librt is used for timer_create in the unit tests for lock tracking (mutex-test.c++).
             ./super-test.sh quick ${{ matrix.compiler }} cpp-features "${{matrix.features}}" extra-libs "-lrt"
-  ManyLinux:
-    runs-on: ubuntu-20.04
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - cross-compiler: manylinux2014-x64
-          - cross-compiler: manylinux2014-x86
-            docker-run-args: --platform linux/386
-    steps:
-      - uses: actions/checkout@v2
-      - name: install dockcross
-        run: |
-            docker run ${{ matrix.docker-run-args }} --rm dockcross/${{ matrix.cross-compiler }} > ./dockcross
-            chmod +x ./dockcross
-      - name: super-test
-        run: |
-            ./dockcross ./super-test.sh quick g++
+  # ManyLinux:
```

Comment:
I'd be fine with deleting these commented builds, I suspect it'll be years before we try to support them again, if ever.

Reply:
Agreed; I deleted the commented-out builds.